2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,23 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  function todoList(todos) {
+  let content = document.querySelector("#content");
+  let unorderedList = document.createElement("#ul");
```

Comment:
The same problem as in the previous exercise
![image](https://user-images.githubusercontent.com/54252969/194708322-158c8f72-0cf8-4cf0-a1cf-259988bfe42b.png)

Please finish this exercise as well.